package.json

```diff
@@ -42,7 +42,7 @@
     "stylelint-tape": "3.0.0"
   },
   "peerDependencies": {
-    "stylelint": ">= 11 < 16"
+    "stylelint": ">= 11 < 17"
```

Comment:
The new plugin structure works on `>=11 <15`? @ybiquitous

Reply:
It's not a **new** structure.

Reply:
OK, so it's still compatible here, right?

Reply:
Yes. `export { ruleName }` happens to work.

Reply:
https://github.com/csstools/stylelint-use-logical/pull/22/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R146

@ybiquitous It is not exported anymore.

Reply:
@ybiquitous Any thoughts?

Reply:
The Stylelint API was not changed, we should not change this range.

Reply:
> The Stylelint API was not changed, we should not change this range.

I'm just confirming whether current pattern also works on `>=11 <15`.

> We just moved to a correct API which worked before by accident and stop working in Stylelint 16.

That's why I said for compatibility, stylelint should not remove the API, and the API is better than current one IMO.